### ISSUE TITLE
Training with very small datasets

### ISSUE THREAD

**Issue by festinais** [NVlabs/stylegan2-ada-pytorch#255]
Hi, this is not a bug but more of an understanding of whether overfitting happens when training with datasets that contain around 1000-1300 samples. 
Would this stylegan work with datasets of such size in theory? I tried training with this dataset, and the FID metric is converging but not lower than 40 sometimes. 

 Also, any ideas on how to test the authenticity of the generated data are highly appreciated. 

Thank you!

**Comment by ellonde** [comment]
Im no expert but cant you just look at the fakes that are produces while training? or try out your network after training?
